src/niveristand/clientapi/_datatypes/rtprimitives.py

```diff
@@ -387,7 +387,11 @@ def value(self, newvalue):
         _DefaultGatewayFactory.get_workspace2().set_single_channel_value(self._channel_name, newvalue)
 
     def _to_data_value(self, value):
-        return ClientApiDoubleValue(value)
+        # in pythonnet 3.0.0+, strings are no longer coerced to 0 by ClientApiDoubleValue
+        if isinstance(value, str):
+            return 0
```

Comment:
Should this be `return ClientApiDoubleValue(0)`?

Reply:
Commit 5876bec5d04db5a75db375378d5329aa2735670e